### DISCUSSION TITLE
TS SYNTAX

### DISCUSSION THREAD

**Discussion by arunbm123** [community/community#128619] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi

export class RequiredError extends Error {
    name: "RequiredError" = "RequiredError";
    constructor(public field: string, msg?: string) {
        super(msg);
    }
}

What this syntax of class Name property type

Kindly some one be kind to explain

**Comment by ebndev** [comment]
Hi @arunbm123, thanks for being a part of the GitHub Community!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from specific project repo, is that right?

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

If you don't mind, would you mind letting us know how you ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your Discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board. Any insight would be helpful, thanks!

**Comment by arunbm123** [comment]
hi
Yes I am looking for help on particular Project.

please find the Image below , I clicked on send Feedback..I thought the
feedback will go to the
person with this Git A/c

[image: image.png]

On Tue, Jun 18, 2024 at 3:58 AM ebndev ***@***.***> wrote:

> Hi @arunbm123 <https://github.com/arunbm123>, thanks for being a part of
> the GitHub Community!
>
> The Profile category is for conversations related to your GitHub profile
> page. I have moved this post to General, however, it looks like you're
> looking for support from specific project repo, is that right?
>
> We want to make sure you’re getting the best support you can and that your
> message goes where you intended, but this may not be the right place.
>
> If you don't mind, would you mind letting us know how you ended up in the Profile
> category
> <https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+>
> when creating your Discussion? We are trying to investigate if there is a
> bug or link that may be re-directing users from the project's Discussions
> board. Any insight would be helpful, thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/128619#discussioncomment-9799515>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEDCWAIPSBMCD7F36E5KHKDZH5PKJAVCNFSM6AAAAABJLOB4NKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOJZGUYTK>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*- Thanks*
*  Arun*
*  Mobile : 9448420662*

**Reply by ebndev** [reply]
Thanks for taking the time to respond! Would you mind uploading the image by responding to this comment in the browser? I am unable to see it when you replied by email. 

https://github.com/orgs/community/discussions/128619#discussioncomment-9801642

**Comment by arunbm123** [comment]
Ok I will upload ...tomarow

**Reply by ebndev** [reply]
@arunbm123 no need, I found what you were talking about. Very appreciative of your help!

**Comment by arunbm123** [comment]
Glad it worked.
